controllers/managedocs_controller.go

```diff
@@ -68,44 +69,47 @@ const (
-	alertRelabelConfigSecretKey        = "alertrelabelconfig.yaml"
-	onboardingValidationKeySecretName  = "onboarding-ticket-key"
-	convergedDeploymentType            = "converged"
-	consumerDeploymentType             = "consumer"
-	providerDeploymentType             = "provider"
-	rhobsRemoteWriteConfigIdSecretKey  = "prom-remote-write-config-id"
-	rhobsRemoteWriteConfigSecretName   = "prom-remote-write-config-secret"
+	managedOCSName                       = "managedocs"
+	storageClusterName                   = "ocs-storagecluster"
+	prometheusName                       = "managed-ocs-prometheus"
+	prometheusServiceName                = "prometheus"
+	alertmanagerName                     = "managed-ocs-alertmanager"
+	alertmanagerConfigName               = "managed-ocs-alertmanager-config"
+	dmsRuleName                          = "dms-monitor-rule"
+	storageSizeKey                       = "size"
+	storageUnitKey                       = "unit"
+	onboardingTicketKey                  = "onboarding-ticket"
+	storageProviderEndpointKey           = "storage-provider-endpoint"
+	enableMCGKey                         = "enable-mcg"
+	notificationEmailKeyPrefix           = "notification-email"
+	onboardingValidationKey              = "onboarding-validation-key"
+	deviceSetName                        = "default"
+	storageClassRbdName                  = "ocs-storagecluster-ceph-rbd"
+	storageClassCephFSName               = "ocs-storagecluster-cephfs"
+	deployerCSVPrefix                    = "ocs-osd-deployer"
+	ocsOperatorName                      = "ocs-operator"
+	mcgOperatorName                      = "mcg-operator"
+	egressNetworkPolicyName              = "egress-rule"
+	egressFirewallName                   = "default"
+	ingressNetworkPolicyName             = "ingress-rule"
+	cephIngressNetworkPolicyName         = "ceph-ingress-rule"
+	providerApiServerNetworkPolicyName   = "provider-api-server-rule"
+	prometheusProxyNetworkPolicyName     = "prometheus-proxy-rule"
+	monLabelKey                          = "app"
+	monLabelValue                        = "managed-ocs"
+	rookConfigMapName                    = "rook-ceph-operator-config"
+	k8sMetricsServiceMonitorName         = "k8s-metrics-service-monitor"
+	alertRelabelConfigSecretName         = "managed-ocs-alert-relabel-config-secret"
+	alertRelabelConfigSecretKey          = "alertrelabelconfig.yaml"
+	onboardingValidationKeySecretName    = "onboarding-ticket-key"
+	convergedDeploymentType              = "converged"
+	consumerDeploymentType               = "consumer"
+	providerDeploymentType               = "provider"
+	rhobsRemoteWriteConfigIdSecretKey    = "prom-remote-write-config-id"
+	rhobsRemoteWriteConfigSecretName     = "prom-remote-write-config-secret"
+	csiKMSConnectionDetailsConfigMapName = "csi-kms-connection-details"
+	desiredODFSubscriptionChannel        = "stable-4.11"
+	odfOCMBundleName                     = "odf-operator"
```

Comment:
@ezio-auditore There was a question in my last review if this is a package name or a bundle name and you introduced a change to rename it as a bundle name (instead of answering the question). But from the code I can see this is referring to a package name in the subscription. Also, I mentioned in my last review OCM instead of OLM and that was a mistake.

Taking all of that into account this should be named `odfOLMPackageName`

Reply:
Sorry, I might have mistook your question. Have updated it accordingly